### DISCUSSION TITLE
Having trouble with creating events

### DISCUSSION THREAD

**Discussion by VolcanoDiver** [hasura/graphql-engine#9662] [Question]
i try to create event trigger with hasura console but it keep says that
"Event trigger with name "<EVENT NAME>" already exists"

but i cannot find that event trigger with hasura console or even in DB (postgre).
(not in metadata too)

how do i find it or delete it and recreate?

**Comment by meetzaveri** [comment]
Hey @VolcanoDiver 

Are you using Hasura Cloud? If yes, then we would recommend creating a support ticket from hasura cloud dashboard so that we can take it forward from there.
